From fe2970c48462ba073476a6997beed5e60d098cef Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Thu, 13 Jun 2024 10:27:19 +0000
Subject: [PATCH 24/24] rdk/snr: change "+=" to "=" for KBUILD_EXTRA_SYMBOLS

ERROR: modpost:
rdk/klm/qat/qat/drivers/crypto/qat/qat_common/intel_qat:
'adf_cfg_dev_add' exported twice. Previous export was in
rdk/klm/qat/qat/drivers/crypto/qat/qat_common/intel_qat.ko

In Makefile of usdm, duplicate settings was added to KBUILD_EXTRA_SYMBOLS.

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/qat/usdm/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/klm/qat/usdm/Makefile b/klm/qat/usdm/Makefile
index 8078d30..4a7fa62 100644
--- a/klm/qat/usdm/Makefile
+++ b/klm/qat/usdm/Makefile
@@ -87,7 +87,7 @@ endif
 
 ifeq ($(ICP_QDM_IOMMU), 1)
     EXTRA_CFLAGS += -DICP_QDM_IOMMU
-    KBUILD_EXTRA_SYMBOLS+=$(QAT_PATH)/Module.symvers
+    KBUILD_EXTRA_SYMBOLS=$(QAT_PATH)/Module.symvers
     export KBUILD_EXTRA_SYMBOLS
 endif
 
-- 
2.39.3

